test/acceptance/behave/README.md

```diff
@@ -1,44 +1,64 @@
-- pip installed 1.4.1 or higher (http://docs.python-guide.org/en/latest/starting/install/linux/)
-- virtualenv installed 1.10.1 or higher (pip install virtualenv) (optional).
+* Ensure you have the following requirements in your system:
+  * Python 2.7. If you need to keep different version of Python interpreter at the same time, you could have a look
+    to [SCL](https://www.softwarecollections.org).
+  * Python pip 1.4.1 or higher
+  * virtualenv 1.10.1 or higher
+  * Requeriments for the fabric Python module (installed in a next step), typically installed using: `yum install gcc python-devel`. [Fabric](http://www.fabfile.org) is a library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
+* Creating a virtual env (e.g. named orion_bh). Actually this step is optional (if you know what a virtual env is, then you would know which 
+  steps to skip in order to avoid using it ;)
 
-Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+```
+virtualenv orion_bh
+```
 
+* Activate virtual env (you can later run `deactivate` to exit out of the virtual env)
 
-#### Requirements to fabric (http://www.fabfile.org/)
-Fabric is a Python (2.5-2.7) library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
-Some requirements are neccesary verify before to install `Fabric` library, mainly to pycripto library (Fabric dependency). 
-This libraries will be install into `requirements.txt`
 ```
-     yum install gcc python-devel
+source orion_bh/bin/activate
 ```
-   
-
-#### Environment preparation:
-
-- If you are going to use a virtual environment (optional):
-  * Create a virtual environment somewhere, `virtualenv venv_name` (optional)
-  * Activate the virtual environment `source venv_name/bin/activate` (optional)
-  * Remember to unset the virtual environment when you're done testing, if it has been previously activated `deactivate` (optional)
-- You may need to set `export GIT_SSL_NO_VERIFY=true` environment variable in your machine
-- Both if you are using a virtual environment or not:
-  * Change to the test/acceptance/behave folder of the project.
-  * Install the requirements for the acceptance tests in the virtual environment
-  * You should add `--upgrade` (if you have a previous installed version of this library)
+
+* Install requirements. Assuming that you are at the test/acceptance/behave directory, run:
+
 ```
-     pip install [--upgrade] -r requirements.txt --allow-all-external
+pip install -r requirements.txt --allow-all-external
 ```
 
+* Set properties.json
+
+```
+cp properties.json.base properties.json
+vi properties.json
+# Ensure that CB_LOG_FILE and CB_PID_FILE are ok in your environment
+# Ensure that CB_EXTRA_OPS
```

Comment:
if the host is remote, ensure that CB_FABRIC_USER, CB_FABRIC_PASS or CB_FABRIC_CERT and CB_FABRIC_SUDO